inventory/service/group_vars/proxy.yaml

```diff
@@ -16,7 +17,7 @@ proxy_backends:
 
   - name: "dashboard"
     options:
-      - "httpchk"
+      - "httpchk HEAD /api/health"
```

Comment:
shouldnt it be GET instead of HEAD method ? 
https://grafana.com/docs/grafana/latest/http_api/other/#returns-health-information-about-grafana

Reply:
HEAD works as well (check with curl) and an advantage of HEAD is that doesn't need to send data back. In this particular case it is not really that much relevant though. For alerta and graphite I explicitly used GET (HEAD works also) to prevent potential caching of data (to really force alerta produce result). For Grafana I thought it was not necessary. But we can surely use GET

Reply:
ok, I see...
on the other hand (not sure you are aware) there is another API endpoint /healthz which is undocumented, but works...it sends back simple OK instead of the version of grafana.

Reply:
not a fan of relying on undocumented APIs, even though /healthz seems to be more a standard